.github/workflows/ci.yml

```diff
@@ -27,6 +27,7 @@ jobs:
             config: "release"
           - os: ubuntu-24.04-arm
             config: "release"
+            dotnet_version: "9.x" # Use .NET 9.x on ARM https://github.com/dotnet/runtime/issues/104123
```

Comment:
Instead I think we should just set the version based on the os name and not bother trying to create a variable.